.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -1,20 +1,50 @@
 name: E2E tests ondemand
 
 on:
+  schedule:
+    # every Mon/Wed/Fri at 08:00 UTC
+    - cron: 0 8 * * 1,3,5
   workflow_dispatch:
     inputs:
       target:
         description: 'Target branch to run E2E tests over'
         required: true
-        # TODO: @andriisoldatenko revisit it
         default: 'release-1.1'
 
 permissions:
   checks: write
 
+# based on:
+# https://docs.dynatrace.com/docs/setup-and-configuration/technology-support/support-model-for-kubernetes
+# https://kubernetes.io/releases/
 jobs:
-  run-in-k8s-1-21:
-    name: Run in Kubernetes 1.21
+  run-matrix:
+    name: Run in Kubernetes/OCP
+    strategy:
+      matrix:
+        include:
+          # k8s
+          - flc_env: dto-k8s-1-24
+            flc_ns: dto-k8s-ondemand
+          - flc_env: dto-k8s-1-25
+            flc_ns: dto-k8s-ondemand
+          - flc_env: dto-k8s-1-26
+            flc_ns: dto-k8s-ondemand
+          - flc_env: dto-k8s-1-27
+            flc_ns: dto-k8s-ondemand
+          - flc_env: dto-k8s-1-28
+            flc_ns: dto-k8s-ondemand
+          - flc_env: dto-k8s-1-29
+            flc_ns: dto-k8s-ondemand
+          # OCP
+          - flc_env: dto-ocp-4-10
+            flc_ns: dto-ocp-ondemand
+          - flc_env: dto-ocp-4-11
+            flc_ns: dto-ocp-ondemand
+          - flc_env: dto-ocp-4-12
+            flc_ns: dto-ocp-ondemand
+          - flc_env: dto-ocp-4-13
+            flc_ns: dto-ocp-ondemand
```

Comment:
:heart: Love it if you can make it work like this

Reply:
yeah it works